src/components/Header/utils/headerUtils.ts

```diff
@@ -19,6 +19,7 @@ const authServerBase = {
   development: "https://dev-login.nypl.org/auth",
 };
 const baseLoginLinks = {
+  // TODO:
   catalog: {
     production: "https://browse.nypl.org/iii/encore/myaccount",
     development: "https://nypl-encore-test.nypl.org/iii/encore/myaccount",
```

Comment:
@nonword would you happen to know the catalog account links?

Reply:
We've been using `https://borrow.nypl.org/?openAccount=checkouts`. There is no dev env yet, so use prod for both.